src/app/navbar/navbar.component.html

```diff
@@ -1,22 +1,48 @@
 <div class="container-fluid">
   <div class="sidebar" [@toggleNav]="state">
-    <mat-toolbar color="primary">
+    <mat-toolbar id="navbar">
       <button type="button" class="btn btn-default btn-lg toggle-button" aria-label="Left Align" (click)="toggle()">
         <span class="glyphicon glyphicon-align-justify" aria-hidden="true"></span>
       </button>
+      <img src="/assets/img/leftBanner.png" id="banner">
     </mat-toolbar>
 
     <ul class="nav text-center">
       <div class="profile">
-        <img  [src]="proPic" onError="this.src='assets/img/profilepic.png';" class="profile-pic img-circle">
+        <div id="edit">
+          <mat-icon>edit</mat-icon>
+        </div>
+        <img [src]="proPic" onError="this.src='assets/img/profilepic.png';" class="profile-pic img-circle">
         <p> {{username}} </p>
       </div>
-      <li><a routerLink="/introduction">{{language?.links.introduction}}</a></li>
-      <li><a routerLink="/malaria-101/activity-1-1">{{language?.links.malaria101}}</a></li>
-      <li><a routerLink="/meds-n-labels/activity-1">{{language?.links.medsAndLabels}}</a></li>
-      <li><a routerLink="/prevent">{{language?.links.prevention}}</a></li>
-      <li><a routerLink="/introduction/activity-1">{{language?.links.pcpolicy}}</a></li>
-      <li><a (click)="toggleInfokit()">{{language?.links.infokit}}</a></li>
+      <li>
+        <a routerLink="/menu" routerLinkActive="active-link" class="menu-link">{{language?.links.menu}}</a>
+      </li>
+      <br>
+      <li>
+        <a class="menu-link" routerLink="/meds-n-labels/activity-3" routerLinkActive="active-link">{{language?.links.doctorChat}}</a>
+      </li>
+      <br>
+      <li>
+        <a class="menu-link" routerLink="/introduction"
+           routerLinkActive="active-link">{{language?.links.introduction}}</a>
+      </li>
+      <br>
+      <li>
+        <a class="menu-link" routerLink="/malaria-101/activity-1-1" routerLinkActive="active-link">{{language?.links.malaria101}}</a>
+      </li>
+      <br>
+      <li>
+        <a class="menu-link" routerLink="/prevent/" routerLinkActive="active-link">{{language?.links.prevention}}</a>
+      </li>
+      <br>
+      <li>
+        <a class="menu-link" routerLink="/meds-n-labels/activity-1" routerLinkActive="active-link">{{language?.links.medsAndLabels}}</a>
```

Comment:
I don't think links should be a part of language file.

Reply:
Actually, this was the way it had been implemented last year. But they are names of the link, they'll differ in different languages. I guess not a bad idea to have it like that. @shreyans29 @Raxerz

Reply:
I don't think there is any harm in having it as it is for now. Maybe we can create an issue later and track it there.

Reply:
sure :)